modules/library/render/src/test/java/org/geotools/renderer/lite/gridcoverage2d/GridCoverageRenderingOutOfViewTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2016, Jumbletree Ltd

```

Comment:
Hi @bencaradocdavies and @jodygarnett

Please find attached my signed contribution agreement.

Thanks

Iain


On 15/06/2016 10:43 a.m., Ben Caradoc-Davies wrote:
>
> In 
> modules/library/render/src/test/java/org/geotools/renderer/lite/gridcoverage2d/GridCoverageRenderingOutOfViewTest.java 
> <https://github.com/geotools/geotools/pull/1212#discussion_r67071493>:
>
> > @@ -0,0 +1,101 @@
> > +/*
> > + *    GeoTools - The Open Source Java GIS Toolkit
> > + *    http://geotools.org
> > + *
> > + *    (C) 2016, Jumbletree Ltd
>
> @matchami <https://github.com/matchami> there is a problem with this 
> copyright header. To contribute code to the GeoTools project, 
> copyright must be assigned to OSGeo. Source code files must use the 
> following header:
> http://docs.geotools.org/latest/developer/introduction.html#source-license
>
> As you are adding a new file, please submit a code contribution 
> agreement. If this work is owned by Jumbletree Ltd, then use the 
> corporate agreement signed by an authorised person:
> http://docs.geotools.org/latest/developer/procedures/contribution_license.html
>
> Please also copy @jodygarnett <https://github.com/jodygarnett> (OSGeo 
> project officer) when you submit your agreement.
>
> Sorry for not spotting this earlier.
>
> I also urge you to squash these commits into a single commit for 
> clarity. You can do this with |git cherry-pick -n COMMIT_ID ...| to 
> prepare your changes and then combine them into a single commit.
>
> I also see that you are working on master in your fork. One of the big 
> changes in Git (from svn) is that branches are super-easy to create 
> and use, and better than working on master. For example, it is easy to 
> create a branch |geot-5421-master|, make your changes as a single 
> commit, push to your GitHub fork, and make a pull request. This allows 
> you to work on your next issue while your pull request is outstanding.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/geotools/geotools/pull/1212/files/1bf3670edbfc34d2332bf1ec017f8eef0260a305#r67071493>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AChibFtTsbf5ewOn8UqnXbd959mQVNjyks5qLy6GgaJpZM4I10aG>.
>

-- 
Sent from my ZX80



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus